a11y-input/src/ally-input.js

```diff
@@ -12,6 +12,14 @@ export class A11yInput extends LitElement {
     super();
     this.label = '';
   }
+
+  // eslint-disable-next-line class-methods-use-this
+  render () {
+    return html`
+      <slot name="label"></slot>
+      <slot name="input"></slot>
+    `;
+  }
```

Comment:
### Added `LitElement` `render` method: 

### Test still fails:

```bash
  a11y input
    ✔ has by default an empty string as label
    ✖ has a static shadowDOM
HeadlessChrome 79.0.3945 (Mac OS X 10.13.6) a11y input has a static shadowDOM FAILED
	AssertionError: expected '<!---->\n      <slot name="label"></slot>\n      <slot name="input"></slot>\n    <!---->' to equal '\n      <slot name="label"></slot>\n      <slot name="input"></slot>\n    '
	    at Context.<anonymous> (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/a11y-input/test/ally-input.test.js:31:1)
Finished in 0.004 secs / 0.006 secs @ 11:10:14 GMT-0500 (Central Daylight Time)

SUMMARY:
✔ 1 test completed
✖ 1 test failed

FAILED TESTS:
  a11y input
    ✖ has a static shadowDOM
      HeadlessChrome 79.0.3945 (Mac OS X 10.13.6)
    AssertionError: expected '<!---->\n      <slot name="label"></slot>\n      <slot name="input"></slot>\n    <!---->' to equal '\n      <slot name="label"></slot>\n      <slot name="input"></slot>\n    '

      + expected - actual

      -<!---->
      +
             <slot name="label"></slot>
             <slot name="input"></slot>
      -    <!---->
      +

    at Context.<anonymous> (/var/folders/j0/0nbp6gns09v_ygsdtbf5dwrw402nfr/T/_karma_webpack_/webpack:/a11y-input/test/ally-input.test.js:31:1)
TOTAL: 1 FAILED, 1 SUCCESS

=============================== Coverage summary ===============================
Statements   : 100% ( 5/5 )
Branches     : 100% ( 0/0 )
Functions    : 100% ( 3/3 )
Lines        : 100% ( 5/5 )
================================================================================
error Command failed with exit code 1.
```

Reply:
#### Note: `lit-html` adds the `<!---->` tags to mark dynamic content that can be updated.